PhysicsTools/PatAlgos/plugins/PATMuonProducer.cc

```diff
@@ -1094,30 +1096,55 @@ void PATMuonProducer::embedHighLevel(pat::Muon& aMuon,
                                      bool primaryVertexIsValid,
                                      reco::BeamSpot& beamspot,
                                      bool beamspotIsValid) {
+  // Generic variable to store measurements
+  std::pair<bool,Measurement1D> result;
+
```

Comment:
for consistency, it seems like ```double d0_corr , d0_err``` should be defined here as well.
This way no changes will be necessary on what's now L1123-1126